### ISSUE TITLE
[Feature] Additional plugins: Slack integration

### ISSUE THREAD

**Issue by tcastelli** [juliancwirko/s-chat-app#9]
Hi there, thanks for this package, I was working on something pretty similar and since I am still on early stages, I was thinking about contributing to this project instead.

More precisely, I was interested in having the chats with some users(admins) also connected with a slack channel. This seems something more like a feature to implement in a plugin, and  would like to know your opinion about what's the best approach in this case, or if it could simply end as a core feature.

**Comment by juliancwirko** [comment]
Hi, I don't know much about Slack integrations. But we would probably need to listen to Slack calls and then update the database. This should be enough and it is probably doable. The problem is that there isn't simple plugin system in simplechat.support yet. 
For now you could clone the app and host it by yourself.

**Comment by tcastelli** [comment]
Yes, I have searched about how this works normally and the standard technique is to set up a bot on your slack account that retrieves and pushes messages back and forth to desired channels, it shouldn't be too complicated to add to the server part if we come from the perspective of the bot being already created and configured in settings.json. 
The "hard" part would be to create a "sync with slack" button on the admin panel that logs into your slack account and creates the bot on your behalf and sets everything nicely.

And yes, I was planning on mixing the code from the admin app with mine since there many options that are not used/thought differently, but since I'm planning to add this feature to my app I thought it would make sense to also give it to your project in any kind of shape (PR to admin app or meteor package for a plugin). From what I can see, I think the easiest for now would be a PR to the main app and then if you think about another way of integrating extra features create a separate meteor package.

**Comment by juliancwirko** [comment]
yeah, sounds good for me :)
